libs/auth-api-lib/src/lib/resources/admin/admin-scope.service.ts

```diff
@@ -113,10 +113,34 @@ export class AdminScopeService {
       throw new BadRequestException(`Scope name "${input.name}" already exists`)
     }
 
+    // We need to manually validate displayName and description, since we reuse the same dto for both create and publish
```

Comment:
This API doesn't have any concept of "publish". This PR is reusing the DTO for post (create) and patch (update).

Reply:
Removed comment